code/modules/mining/machine_processing.dm

```diff
@@ -5,6 +5,36 @@
 /obj/machinery/mineral
 	var/input_dir = NORTH
 	var/output_dir = SOUTH
+	var/turf/input_turf = null
+	processing_flags = START_PROCESSING_ON_INIT | FAST_PROCESS_SPEED
+
+/obj/machinery/mineral/Initialize(mapload)
+	. = ..()
+	register_input_turf()

```

Comment:
This will already get called due to its parent calling begin_processing()

Reply:
Ah yeah I see what you mean, I think I'm going to just go ahead and make all the mineral type machines use manual start. The way I have it right now with *just* the ORM converted kinda screws up the other machines.